torch/optim/asgd.py

```diff
@@ -131,6 +141,7 @@ def step(self, closure=None):
                 foreach=group["foreach"],
                 maximize=group["maximize"],
                 differentiable=group["differentiable"],
+                capturable=group["capturable"],
```

Comment:
A general comment: we'd have to propagate the capturable argument across all our APIs for ASGD including single_tensor and publicize it--this would fix the CI.

This would include adding the correct documentation and checks similar to NAdam, AdamW and Adam.